usaspending_api/download/tests/integration/test_download_accounts.py

```diff
@@ -497,3 +497,28 @@ def test_empty_array_filter_fail(client, download_test_data):
     assert (
         "Field 'filters|def_codes' value '[]' is below min '1' items" in resp.json()["detail"]
     ), "Incorrect error message"
+
+
+@pytest.mark.django_db(databases=[settings.DOWNLOAD_DB_ALIAS, settings.DEFAULT_DB_ALIAS])
+def test_file_c_spark_download(client, download_test_data):
```

Comment:
Is this actually hitting the Spark download? I don't see usage of the experimental flag so I believe this should be hitting Postgres.